Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 
+gem 'aruba'
```

Comment:
Aruba is only needed in the homework_tester directory, please remove this and the Gemfile.lock changes...